packages/core/react/src/useWallet.ts

```diff
@@ -5,20 +5,18 @@ import {
     Wallet,
     WalletAdapterProps,
     WalletName,
+    WalletReadyState,
 } from '@solana/wallet-adapter-base';
 import { createContext, useContext } from 'react';
 
-export interface EnhancedWallet extends Wallet {
-    ready: boolean;
+export interface WalletWithReadyState extends Wallet {
+    readyState: WalletReadyState;
 }
-
-export interface WalletContextState extends Omit<WalletAdapterProps, 'ready'> {
-    wallets: EnhancedWallet[];
-    walletsByName: Record<WalletName, EnhancedWallet>;
```

Comment:
Took a chance and nerfed `walletsByName` from the context.

Reply:
Yeah, that's the right call, we were exposing too much.